Assets/MixedRealityToolkit/Providers/UnityInput/MouseController.cs

```diff
@@ -65,32 +65,14 @@ public void Update()
                 return;
             }
 
-            if (InputSource.Pointers[0].BaseCursor != null)
-            {
-                controllerPose.Position = InputSource.Pointers[0].BaseCursor.Position;
-                controllerPose.Rotation = InputSource.Pointers[0].BaseCursor.Rotation;
-            }
-
-            mouseDelta.x = -UInput.GetAxis("Mouse Y");
-            mouseDelta.y = UInput.GetAxis("Mouse X");
-            MixedRealityToolkit.InputSystem?.RaiseSourcePositionChanged(InputSource, this, mouseDelta);
-            MixedRealityToolkit.InputSystem?.RaiseSourcePoseChanged(InputSource, this, controllerPose);
-            MixedRealityToolkit.InputSystem?.RaiseSourcePositionChanged(InputSource, this, UInput.mouseScrollDelta);
-
             for (int i = 0; i < Interactions.Length; i++)
```

Comment:
We don't seem to be raising events for the Mouse Scroll, Spatial Pointer and Spatial Grip inputs.

Reply:
spatial grip is only there because we make assumptions that only controllers having spatial grip available in the mapping can drag objects. We don't need to actually use that input type to interact with objects. In case of the mouse you have up and down and pointer position events for that. The other two events were already there and i didn't want to remove them from the mapping to not break anything, but both of them were basically not working / doing anything. I'm afraid I'm not familiar enough with the input system or what the guideline is on now used mappings. What would I have to  implement for those? Spatial cursor doesn't make any sense with mouse, scroll neither with our current implementation as the mouse cursor is snapping to interactable objects and one could argue you could move the camera with mouse wheel, but that code shouldn't be in the mouse pointer / controller I suppose?!